tests/benchmarks/test_multiplexer.py

```diff
@@ -9,10 +10,17 @@
 IP_ADDR = ipaddress.ip_address("8.8.8.8")
 
 
-def test_process_1000_2048_byte_channel_messages(
+@pytest.mark.parametrize(
+    ("size", "message_count"),
+    [(2048, 1000), (1024 * 1024, 100)],
+    ids=["1000@2KiB", "250@1MiB"],
+)
+def test_multiplex_channel_message(
     benchmark: BenchmarkFixture,
     multiplexer_client: Multiplexer,
     multiplexer_server: Multiplexer,
+    size: int,
+    message_count: int,
 ) -> None:
     """Test writing 1000 2048 byte messages to the channel and reading them back."""
```

Comment:
Update the docstring.

Reply:
fixed in 98206e1b8c6900d172a7b373a7c1823509f8dbff